.github/workflows/acceptancetest.yml

```diff
@@ -0,0 +1,25 @@
+name: acceptancetest
```

Comment:
Maybe just `acceptance` is enough? @strimzi/maintainers wdyt?

Reply:
I am fine, but we should change the YAML file name as well then.